desktop-exporter/app/components/traceList.tsx

```diff
@@ -13,73 +13,79 @@ import { useSize } from "@chakra-ui/react-use-size";
 
 import { TraceSummary } from "../types/api-types";
 
- const listItemHeight = 100;
- const listItemTopMargin = 10;
+const listItemHeight = 100;
+const listItemBottomMargin = 2;
 
- type RowProps = {
-   index: number;
-   style: Object;
-   data: TraceSummary[];
- };
+type RowProps = {
+  index: number;
+  style: Object;
+  data: TraceSummary[];
+};
 
- function Row({ index, style, data }: RowProps) {
-   let isSelected = useMatch(`traces/${data[index].traceID}`);
+function Row({ index, style, data }: RowProps) {
+  let isSelected = useMatch(`traces/${data[index].traceID}`);
+  let backgroundColour = isSelected
+    ? useColorModeValue("whiteAlpha.800", "whiteAlpha.400")
+    : useColorModeValue("whiteAlpha.600", "whiteAlpha.200");
+  let borderLeft = isSelected ? "7px solid" : "none";
 
-   let backgroundColour = isSelected
-     ? useColorModeValue("teal.100", "teal.700")
-     : useColorModeValue("whiteAlpha.700", "whiteAlpha.400");
+  return (
+    <div style={style}>
+      <LinkBox
+        bgColor={backgroundColour}
+        height="100px"
+        padding="10px"
+        marginBottom={`${listItemBottomMargin}px`}
```

Comment:
I think if it's pixels you can just write the value without the string formatting:
```suggestion
        marginBottom={listItemBottomMargin}
```

Reply:
Chakra interprets just the value as 0.5 rem, and I need the height of the row to stay predictable for react-window's sake.